app/assets/javascripts/premadeModal.js

```diff
@@ -0,0 +1,22 @@
+var onReadyStrategiesModal = function() {
+
+	var modal = document.getElementById('myModal');
+	var btn = document.getElementById("myBtn");
+	var span = document.getElementsByClassName("close")[0];
+	
+	if(modal || btn || span){
+		btn.onclick = function() {
+		    modal.style.display = "block";
```

Comment:
hmmmmmm but this also applies to when the modal is present on the page if a modal is already open it still applies to being available on the page, which is what I am checking for.